glmmTMB/src/glmmTMB.cpp

```diff
@@ -772,6 +775,19 @@ Type objective_function<Type>::operator() ()
         SIMULATE {
           yobs(i) = glmmtmb::rtweedie(s1, s2, s3);
         }
+      case lognormal_family:
+	// FIXME: documentation??
+	// parameterized in terms of mean and SD on *data* scale, i.e.
+	// mu = exp(logmu + logsd^2/2)
+	// sd = sqrt((exp(logsd^2)-1)*exp(2*logmu + logsd^2)) = mu*sqrt(exp(logsd^2)-1)
+	// 1+(sd/mu)^2 = exp(logsd^2)
+	// logsd = sqrt(log(1+(sd/mu)^2))
+	// logmu = log(mu)- 
+        s1 = log(1 + pow(phi(i)/mu(i), 2.0));
+        s2 = sqrt(s1);
```

Comment:
I think `s2` is the sd of the normal distribution, so it should be the 3rd argument to `dnorm`. 

I think `s3` agrees with what I have for the mean of the normal distribution in [a related application](https://github.com/mebrooks/StockRecruit/blob/b0ffb731c24a1ad6133829ac11194bb1353bb53f/StockRecruitSET/R/simulateMethods.R#L45) that I thought very carefully about, many years ago.